client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const requestId = siteId => `pma-link-request-${ siteId }`;
+
+export const requestPmaLink = siteId =>
+	requestHttpData(
+		requestId( siteId ),
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/hosting/pma`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			fromApi: () => ( { pmaUrl } ) => {
+				return [ [ requestId( siteId ), { pmaUrl } ] ];
+			},
+		}
+	);
+
+const PhpMyAdminCard = ( { translate, siteId, pmaLink, loading } ) => {
+	useEffect( () => {
+		if ( pmaLink ) {
+			window.open( pmaLink );
+		}
+	}, [ pmaLink ] );
+
```

Comment:
For me this only works the first time the 'Access PhpMyAdmin' button is clicked, and doesn't do anything subsequent clicks unless page is reloaded. Looks like subsequent  dispatches of requestHttpData aren't caused a change in the store and a rerender of the component. I wouldn't worry about looking too closely into this until we finalise an approach to the phpmyadmin link - but something to come back to if we go with the client side approach (or any other places where we use this pattern).